### DISCUSSION TITLE
HTMX Support ?

### DISCUSSION THREAD

**Discussion by xtox-io** [emmett-framework/emmett#491] [Q&A]
Does anyone have a sample to integrate HTMX with Emmet ?
I might be too lazy and prefer to ask first :)

**Comment by gi0baro** [comment]
@xtox-io if by *support* you mean a ready-to-go integration, then the answer is no :)

In general it should be quite easy to use HTMX in Emmett without any added syntax-sugar; in case you have any specific use-case in mind feel free to ask for more specific points and I'll happily provide examples for those.
